### DISCUSSION TITLE
Synced Lyrics

### DISCUSSION THREAD

**Discussion by Ja4e** [strawberrymusicplayer/strawberry#1630] [Ideas]
It would be better to add this functionality.

**Comment by guitaripod** [comment]
karaoke lyrics would be great. it's the only feature that is missing from this app (from stuff i'm used to with AM)

**Comment by Panelgamer** [comment]
COMPLETELY BASED! We need a Synced Lyrics from our own files.

**Comment by Ja4e** [comment]
we can try use snycedlyrics pythron library to request all those lyrics automatically

**Comment by jonaski** [comment]
Currently not possible, suggest it on: https://forum.strawberrymusicplayer.org/category/6/feature-suggestions

**Reply by guitaripod** [reply]
not possible and software don't go together!

**Comment by guitaripod** [comment]
Here
* [#2055]
